src/Backend/Modules/Slideshows/Layout/Templates/Add.html.twig

```diff
@@ -16,6 +16,13 @@
         </label>
         {% form_field title %} {% form_field_error title %}
       </div>
+      <div class="form-group">
```

Comment:
you should only show/add this dropdown if there are more than 1 possible template, just like we do in the Content Blocks module